src/cartesian.jl

```diff
@@ -1,6 +1,8 @@
 Base.zero(lv::T) where T<:LorentzVector = T(0,0,0,0)
-mass(lv::LorentzVector) = sqrt(dot(lv, lv))
-pt(lv::LorentzVector) = sqrt(muladd(lv.x, lv.x, lv.y^2))
+mass2(lv::LorentzVector) = dot(lv, lv)
+mass(lv::LorentzVector) = mass2(lv) < 0.0 ? sqrt(-mass2(lv)) : sqrt(mass2(lv))
```

Comment:
is this `negative` mass trick universal? what's ROOT's `Math::LorentzVector` behavior on this

Reply:
ROOT adopts this convention for space-like 4-vectors, so that's some precedent

Reply:
Depends what one thinks of as being *wrong*. Negative mass is not ambiguous - but it may be surprising. I'm ok with rolling this back.

Reply:
Yeah, I guess I mean if we error, it's trivial for user to do what they want. But not the other way, they may silently get wrong results

Reply:
I checked in the Python Vector package and it also conforms to this behaviour:

```python
In [1]: import vector

In [2]: ep_vec = vector.obj(x=8.1, y=2.2, z=3.3, E=4.4)

In [3]: ep_vec.mass
Out[3]: -7.87273777030583
```

And I found I missed a `-` sign on the mass method, oops! Fixed in incoming commit.